README.md

```diff
@@ -1,13 +1,88 @@
-# QA Todo
-- Update ReadMe accordingly with all candidate information. This is a candidate facing repo.
-- Add playwright setup
-- Make repo private
+# Qarchitect Technical Assignment
+Thanks for applying! The steps of this interview process are as follows:
+1. Phone screening
+2. Technical assessment <- this is where we are now!
+3. Technical interview
+4. Cultural interviewv
+5. C-suite interview
+
+## Technical Assignment Details
+This assessment is meant to imitate what a working environment is like. You are welcome to use all the tools available to you as you would in the role itself. However, should you progress to the next round of interviews you will be expected to perform an in depth explanation of your solutions.
+
+We do not want this to take any more of your time than necessary, so please do yourself a favor and set a maximum 3 hour time limit. 
+
+We have created our own version of TODO MVC. It is the same as the regular one, but we've added a few bugs.
+
+There are 3 main pieces to the homework. Each is described in more detail further in this doc.
+
+3 Pieces of the assignment:
+
+1. Create a test strategy for TODO MVC
+2. Help update/create the existing Playwright tests in this repo 
+3. Find a few of the bugs we have purposely created in TODO MVC and submit bug reports for each
```

Comment:
Do we need to say to submit bug reports through Gitlab issues just to be totally clear? It might make more sense on the submission details.